### PR TITLE
Removed the api key refresh rate error and set it to 60 minutes as de…

### PR DESCRIPTION
**Pull request by rodrigopavezi**
…fault when there no api key and the refresh rate is lower than 60 minutes

@donnapep @stulees please review. cheers

### CODE REVIEW

File: src/widget/components/spreadsheet.jsx [hunk 1]
```diff
@@ -215,24 +215,16 @@ const Spreadsheet = React.createClass({
...
-    } else {
-      stop = true;
-      this.showError("Missing API Key.");
-      this.logEvent({
-            "event": "error",
-            "event_details": "missing API Key",
-            "url": params.spreadsheet.url
-          }, true);
+    } else if (params.spreadsheet.refresh < 60) {
+      sheet.setAttribute("refresh", 60 * 60);
```
Comment (stulees) [inline src/widget/components/spreadsheet.jsx]: @rodrigopavezi Why `60 * 60`? Can you clarify why we don't just set it to `60`?

Comment (rodrigopavezi) [inline src/widget/components/spreadsheet.jsx]: @stulees up here it is doing 60*60 when the settings sends 60. https://github.com/Rise-Vision/widget-google-spreadsheet/pull/200/files#diff-c86e7b870625e705a60e22dec96539eeR204. It should be 1 hour right? cheers

Comment (stulees) [inline src/widget/components/spreadsheet.jsx]: @rodrigopavezi Ah, this widget is still using old release of `rise-google-sheet` right? Okay then. We will need to remember to change that as our new release of `rise-google-sheet` expects refresh value to be in minutes, so we will not want `* 60` when implementing using the new version.

**Comment by stulees** [comment]
@rodrigopavezi 👍

**Pull request merged**